app/controllers/journeys/expenses/advertisingOrMarketing/AdvertisingAmountController.scala

```diff
@@ -57,22 +56,15 @@ class AdvertisingAmountController @Inject() (override val messagesApi: MessagesA
 
   def onSubmit(taxYear: TaxYear, businessId: BusinessId, mode: Mode): Action[AnyContent] = (identify andThen getData andThen requireData) async {
     implicit request =>
-      def handleForm(accountingType: String): Future[Result] =
-        formProvider(request.userType)
-          .bindFromRequest()
-          .fold(
-            formWithErrors => Future.successful(BadRequest(view(formWithErrors, mode, request.userType, taxYear, businessId))),
-            value => handleSuccess(value, AccountingType.withName(accountingType.toUpperCase))
-          )
-      def handleSuccess(value: BigDecimal, accountingType: AccountingType): Future[Result] =
-        selfEmploymentService
-          .persistAnswer(businessId, request.userAnswers, value, AdvertisingOrMarketingAmountPage)
-          .map(updated => Redirect(navigator.nextPage(AdvertisingOrMarketingAmountPage, mode, updated, taxYear, businessId, Some(accountingType))))
-
-      selfEmploymentService.getAccountingType(request.user.nino, businessId, request.user.mtditid) flatMap {
-        case Left(_)               => Future.successful(Redirect(JourneyRecoveryController.onPageLoad()))
-        case Right(accountingType) => handleForm(accountingType)
-      }
```

Comment:
In the controllers where we've removed this call to the service, it would also be good to remove the mocked cals from their corresponding UTs.